### PR TITLE
migrate digest related methods to avoid using channel names

### PR DESCRIPTION
**Pull request by trilopin**
## Summary
Change approach at the digest messages sent by playbooks bot. After this change, they will show run name instead of the channel displayname, and additionally link to the RDP instead of the channel

## Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-playbooks/issues/1571

## Checklist
- [ ] Telemetry updated
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

**Review by m-ripper — APPROVED** [review]
Looks good to me!

**Review by m-ripper — CHANGES_REQUESTED** [review]
Looks good overall, ~~only the skipped test-case needs to be discussed.~~

File: tests-e2e/cypress/integration/playbooks/edit_spec.js [hunk 1]
```diff
@@ -830,14 +830,16 @@ describe('playbooks > edit', () => {
                 });
             });
 
-            it('can fill a channel', () => {
+            it.skip('can fill a channel and is persisted', () => {
```
[the remark this answers is not in the document]
Comment (trilopin) [reply tests-e2e/cypress/integration/playbooks/edit_spec.js]: yes, until it is reliable

**Comment by trilopin** [comment]
I added a couple of helpers to cypress to handle telemetry, let me know if they feel useful. If so, I'll create tickets to migrate other telemetry assertions in e2e.

**Review by m-ripper — APPROVED** [review]

**Review by crspeller — APPROVED** [review]

**Pull request merged**